pyproject.toml

```diff
@@ -145,7 +145,7 @@ lint.select = [
     # pydocstyle
     "D",
 ]
-line-length = 100
+line-length = 105
```

Comment:
i was not able to find a ruff setting to ignore the doctest output within docstrings

so i increased line-length to wave the errors

```shell
src\validators\crypto_addresses\eth_address.py:43:101: E501 Line too long (103 > 100)
```